pkg/convert/go_test.go

```diff
@@ -0,0 +1,67 @@
+package convert
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+var modelMsg = model.Messages{
+	Language: language.English,
+	Messages: []model.Message{
+		{
+			ID:          "1",
+			Message:     "message1",
+			Description: "description1",
+			Fuzzy:       true,
+		},
+		{
+			ID:          "2",
+			Message:     "message2",
+			Description: "description2",
+			Fuzzy:       false,
+		},
+	},
+}
+
+func TestToGo(t *testing.T) {
+	t.Parallel()
+
+	expected := []byte(`{"language":"en","messages":
+[{"id":"1","meaning":"description1","message":"","translation":"message1","fuzzy":true},
+{"id":"2","meaning":"description2","message":"","translation":"message2"}]}`)
```

Comment:
If you want to use `Compact` you could also write fully intended json which is more readable
```suggestion
	expected := []byte(`
	{
		"language":"en",
		"messages":[
			{
				"id":"1",
				"meaning":"description1",
				"message":"",
				"translation":"message1",
				"fuzzy":true
			},
			{
				"id":"2",
				"meaning":"description2",
				"message":"",
				"translation":"message2"
			}
		]
	}`)
```